frontend_tests/puppeteer_lib/common.ts

```diff
@@ -265,12 +265,18 @@ class CommonUtils {
     }
 
     async ensure_enter_does_not_send(page: Page): Promise<void> {
+        let enter_sends = false;
         await page.$eval(".enter_sends_false", (el) => {
             if ((el as HTMLElement).style.display !== "none") {
-                // Click events gets propagated to `.enter_sends` which toggles the value.
-                (el as HTMLElement).click();
+                enter_sends = true;
```

Comment:
This doesn’t work. A variable in the Node.js context cannot be mutated by a function evaluated in the browser context.

- #25196